spec.bs

```diff
@@ -785,11 +826,22 @@ report=] |report|, perform the following steps. They return a [=list=] of
 1. [=list/Append=] |aggregationServicePayload| to |aggregationServicePayloads|.
 1. Return |aggregationServicePayloads|.
 
-To <dfn>obtain the public key for encryption</dfn>, perform an
-[=implementation-defined=] sequence of steps. They return a [=tuple=] consisting
-of a public key and a [=string=] (which should uniquely identify the public
-key), or an error in the event that the [=user agent=] failed to obtain the
-public key.
+To <dfn>obtain the public key for encryption</dfn> given an [=aggregation
+coordinator=] |aggregationCoordinator|, perform the following steps. They return
+a [=tuple=] consisting of a public key and a [=string=], or an error.
```

Comment:
Are we restricting what public keys to use, e.g., [X25519](https://datatracker.ietf.org/doc/html/rfc7748)?
Adding some explainations for public key might be good, such as its [length requirement ](https://crsrc.org/c/content/browser/aggregation_service/public_key.h;drc=abbfc06e9b070c9ea64af2b233624646cef00252;l=39), and a definition for it if possible.

Reply:
Yes, we should -- the current spec is very terse (simply describes getting the key as implementation-defined). This PR expands it to give more specificity about where the public keys are hosted, but we should continue detailing more.

I've expanded the Issue below to reflect this, but I think we can probably defer that actual change to a separate PR.